product_input_interface/allowed_products.py

```diff
@@ -0,0 +1,28 @@
+class WarehouseProducts:
+    __ALLOWED_PRODUCTS_TYPE = ["helmet", "rifle", "medkit"]
```

Comment:
that's basically good but you can read what are the `enums` and how to use them and I would refactor that code to use the enum :)